conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -247,6 +244,23 @@ object DynamicConformanceJob {
     }
   }
 
+  private def handleEmptyOutputAfterConformance(implicit spark: SparkSession): Unit = {
+    val areCountMeasurementsAllZero = Atum.getControMeasure.checkpoints
+      .flatMap(checkpoint =>
+        checkpoint.controls.filter(control =>
+          control.controlName.equalsIgnoreCase(za.co.absa.atum.core.Constants.controlTypeRecordCount)))
+      .forall(m => Try(m.controlValue.toString.toLong).getOrElse(1L) == 0L)
```

Comment:
Can `toString.toLong` be replaced by pattern matching?

Reply:
Might be a lot of things to match since the input is `Any`. `toString()` is safer here in my opinion.

Reply:
Well, not 'safer', more like 'simpler' while covering all cases something can go wrong I can imagine.